src/platforms/common/enriching-events/scopes.mdx

```diff
@@ -140,4 +160,3 @@ launch(SentryContext()) {
   }
 }
 ```
-</PlatformSection>
```

Comment:
This is probably wrong too, Kotlin coroutines is available even for iOS now: https://sentry-docs-git-fork-designedbyz-update-scopes.sentry.dev/platforms/apple/guides/ios/enriching-events/scopes/#kotlin-coroutines

Reply:
Can you clarify? I'm confused about whether you mean the initial platform section was wrong, or if removing it was wrong. I'd think that if coroutines is available on iOS now that it might need mentioned?

Reply:
Sure, I just thought that the whole section of Kotlin Coroutines for iOS was visible because of the removal of L143, maybe it's not the case, but The Kotlin Coroutines should not be available for iOS, We have specific docs for Kotlin only https://docs.sentry.io/platforms/kotlin/
I'd not mix both contents yet while we don't offer official support for Kotlin multiplatform.

Reply:
@marandaneto, got it. Fixed in 70231d38